client-sdk/ts-web/core/reflect-go/go.mod

```diff
@@ -3,59 +3,3 @@ module github.com/oasisprotocol/oasis-sdk/client-sdk/ts-web/core/reflect-go
 go 1.17
 
 require github.com/oasisprotocol/oasis-core/go v0.2103.6
-
-require (
```

Comment:
How did this happen? This is a new thing with Go 1.17 so it should remain.

Reply:
oh I'm on the wrong version of Go locally :facepalm:

Reply:
removing this commit